pkgs/development/libraries/agda/agda-categories/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, mkDerivation, fetchFromGitHub, standard-library }:
+
+mkDerivation rec {
+  version = "0.1";
+  pname = "agda-categories";
+
+  src = fetchFromGitHub {
+    owner = "agda";
+    repo = "agda-categories";
+    rev = "release/v${version}";
+    sha256 = "0m4pjy92jg6zfziyv0bxv5if03g8k4413ld8c3ii2xa8bzfn04m2";
+  };
+
+  buildInputsAgda = [ standard-library ];
+
+  meta = with lib; {
+    inherit (src.meta) homepage;
+    description = "A new Categories library";
+    license = licenses.bsd3;
+    platforms = platforms.unix;
+    hydraPlatforms = [];
```

Comment:
Consider adding a brief explanation of why this package should be excluded from Hydra (and thus excluded from release testing).

Reply:
That makes sense